api/public/cerbos/policy/v1/policy.proto

```diff
@@ -130,7 +130,7 @@ message RolePolicy {
   }
   string scope = 2 [(buf.validate.field).string.pattern = "^([[:alnum:]][[:word:]\\-]*(\\.[[:word:]\\-]*)*)*$"];
   repeated RoleRule rules = 3;
-  ScopePermissions scope_permissions = 4;
+  ScopePermissions scope_permissions = 4 [(buf.validate.field).required = true];
```

Comment:
I think this needs an `enum in` constraint as well. Otherwise, `SCOPE_PERMISSIONS_UNSPECIFIED` would be accepted as a valid value as well.